.ci/sample-objects/aws/md-scale-down.yaml

```diff
@@ -0,0 +1,27 @@
+# Sample machine-deploy object
+
+apiVersion: machine.sapcloud.io/v1alpha1
```

Comment:
While integrating other providers, let's try to move this samples as part of the pre-created test-shoot-clusters.